biome.json

```
@@ -0,0 +1,32 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
+	"vcs": {
+		"enabled": false,
+		"clientKind": "git",
+		"useIgnoreFile": false
+	},
```

高速化のためにも、`.gitignore`をbiomeのignoreとして使った方が便利でいいかなと思いました

```suggestion
	"vcs": {
		"enabled": true,
		"clientKind": "git",
		"useIgnoreFile": true
	},
```